app/views/candidate_interface/equality_and_diversity/edit_ethnic_group.html.erb

```diff
@@ -7,10 +7,13 @@
       <%= f.govuk_error_summary %>
 
       <%= f.govuk_radio_buttons_fieldset :ethnic_group, caption: { text: t('equality_and_diversity.title'), size: 'xl' }, legend: { text: t('equality_and_diversity.ethnic_group.title'), size: 'xl', tag: 'h1' } do %>
+        <p class="govuk-body">Training providers will not see your answer to this question when deciding whether to offer you a place.</p>
+        <p class="govuk-body">They will only see your answer if you accept an offer from them.</p>
+
```

Comment:
The two paragraphs should come before the `govuk_radio_buttons_fieldset`, and a separate legend ("What is your ethnic group?") should be visible - separate from the h1 which is now just "Ethnicity". The caption would have to move too.

Something a bit like this:
```suggestion
        <p class="govuk-body">Training providers will not see your answer to this question when deciding whether to offer you a place.</p>
        <p class="govuk-body">They will only see your answer if you accept an offer from them.</p>
      <%= f.govuk_radio_buttons_fieldset :ethnic_group, legend: { text: "What is your ethnic group?", size: 'm' } do %>

```